cmd/test_load.go

```diff
@@ -240,14 +241,23 @@ func loadAndConfigureTest(
 	return test.consolidateDeriveAndValidateConfig(gs, cmd, cliConfigGetter)
 }
 
+func loadSystemCertPool(gs *globalState) {
+	if _, err := x509.SystemCertPool(); err != nil {
+		gs.logger.WithError(err).Warning("Unable to load system cert pool")
+	}
+}
+
 func (lct *loadedAndConfiguredTest) buildTestRunState(
+	gs *globalState,
```

Comment:
```suggestion
```
`loadadedAndConfiguredTest` contains the `TestPreInitState` where the logger is available, so If we use just the logger then we can avoid adding this dependency.